### PR TITLE
[serve] add cmdline options to control which microbenchmarks to run

### PR DESCRIPTION
**Pull request by zcin**
## Why are these changes needed?

Add command line flags that can be used to control which microbenchmarks to run. The release test pipeline will use `--run-all`, but when running the script for debugging, these flags make it a lot easier to run only the tests you want.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/serve_tests/workloads/microbenchmarks.py [hunk 1]
```diff
@@ -242,8 +286,31 @@ async def _main(output_path: Optional[str]):
...
 @click.command()
 @click.option("--output-path", "-o", type=str, default=None)
-def main(output_path: Optional[str]):
-    asyncio.run(_main(output_path))
+@click.option("--run-all", is_flag=True)
+@click.option("--run-http", is_flag=True)
+@click.option("--run-grpc", is_flag=True)
+@click.option("--run-handle", is_flag=True)
+@click.option("--run-latency", is_flag=True)
+@click.option("--run-throughput", is_flag=True)
```
Comment (edoakes) [inline release/serve_tests/workloads/microbenchmarks.py:294]: what's the default?

Comment (zcin) [reply release/serve_tests/workloads/microbenchmarks.py:294]: defaults are all False

Comment (edoakes) [reply release/serve_tests/workloads/microbenchmarks.py:294]: seems confusing; would suggest default runs everything (or the most common one in isolation)

**Review by edoakes — APPROVED** [review]

File: release/serve_tests/workloads/microbenchmarks.py (hunk 1 above)
Comment (zcin) [reply release/serve_tests/workloads/microbenchmarks.py:294]: Added logic that defaults to run everything if no other flags are set!

**Comment by zcin** [comment]
@edoakes tests are passing, ready to merge

**Pull request merged**